cpu/cc2538/lpm.c

```diff
@@ -73,13 +73,13 @@ static unsigned long irq_energest = 0;
 #define DEEP_SLEEP_PM1_THRESHOLD    10
 #define DEEP_SLEEP_PM2_THRESHOLD    100
 /*---------------------------------------------------------------------------*/
-#define assert_wfi() do { asm("wfi"::); } while(0)
+#define assert_wfi() do { asm ("wfi" ::); } while(0)
 /*---------------------------------------------------------------------------*/
 #if LPM_CONF_STATS
 rtimer_clock_t lpm_stats[3];
 
 #define LPM_STATS_INIT()         do { memset(lpm_stats, 0, sizeof(lpm_stats)); \
-  } while(0)
+} while(0)
```

Comment:
This looks weird. It would be better to move the whole do/while onto its own line indented with two spaces after the `#define` line.